org.kde.keysmith.json

```diff
@@ -0,0 +1,48 @@
+{
+    "id": "org.kde.keysmith",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "keysmith",
+    "rename-icon": "keysmith",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--own-name=org.kde.Keysmith"
```

Comment:
Maybe if you set the app-id to match exactly, you don't need to "own-name". That's probably the best.

Reply:
Do you mean the "id" field or in the application itself?

Reply:
This probably require to rename the appdata file.

Reply:
org.kde.Keysmith for the "id" seems inconsistent with other kde apps, not sure if there are any implications of that

Reply:
the issue here is that by default the app own the dbus name that match the id, so the two should match. Now the question is whether you are sure of the dbus name?

Reply:
Consistency between apps is much less important than matching whatever the app itself expresses - although if we want to be consistent in KDE going into the codebase and changing it isn't *that* hard ;-)